regress/pesterTests/SSH.Tests.ps1

```diff
@@ -131,8 +143,36 @@ Describe "E2E scenarios for ssh client" -Tags "CI" {
             0 | ssh -p $port $ssouser@$server pause
             $true | Should Be $true
         }#>
-    }
+    }    
+    
+    Context "$tC - configure default shell Scenarios" {
+        BeforeAll {$tI=1}
+        AfterAll{$tC++}
+        AfterEach {
+            Remove-ItemProperty -Name $registryKeyName -Path $defaultRegistryPath -ErrorAction SilentlyContinue
+        }
+
+        It "$tC.$tI - default shell as powershell" {
+            $shell_path = (Get-Command powershell.exe -ErrorAction SilentlyContinue).path
+            if($shell_path -ne $null) {
+                ConfigureDefaultShell -default_shell_path $shell_path
+
+                $o = ssh test_target Write-Output 1234
```

Comment:
A better way to test this is the following
$o = ssh test_target (get-process -PID $pid).processname
$o | Should Be "powershell"